README.md

```diff
@@ -58,35 +57,81 @@ steps:
 
 ![Simple Sequence](assets/USS1.png)
 
-For an additional examples, this [Sequence Diagram](assets/USS2.png) shows a
-more complex operation with three USSs, two trying to plan at the same time.
-This [Sequence Diagram](assets/USS3.png) shows a USS affecting multiple grids.
-This [Sequence Diagram](assets/USS4.png) shows another USS updating one of
-multiple grids during an update. And finally, this
-[Sequence Diagram](assets/USS5.png) shows different examples of protection with
-concurrent updates.
+
+## Architecture
```

Comment:
Nearly everything above this point should be removed, or updated to reflect what is in this repo (though an update could happen in a follow-up PR).

Reply:
A lot of it looks like it still applies.. I've removed the sequence diagram for now though. I think I'll leave the general intro to somebody else though and try to stick to the technical components :)

Reply:
done!